python/hsfs/client/exceptions.py

```diff
@@ -62,6 +62,27 @@ class FeatureStoreException(Exception):
     """Generic feature store exception"""
 
 
+class VectorDatabaseException(Exception):
+
+    REQUESTED_K_TOO_LARGE = "REQUESTED_K_TOO_LARGE"
+    REQUESTED_K_TOO_LARGE_INFO_K = "k"
+    REQUESTED_NUM_RESULT_TOO_LARGE = "REQUESTED_NUM_RESULT_TOO_LARGE"
+    OTHERS = "OTHERS"
```

Comment:
maybe somehow separate these variable to make it clear what is used for `reason` and `info`